server/routes/standardRouter.js

```diff
@@ -1,16 +1,12 @@
 const express = require('express')
+const csrfTokenMiddleware = require('../middleware/csrfTokenMiddleware')
 
 module.exports = ({ authenticationMiddleware }) => {
   return (routes) => {
     const router = express.Router()
 
     router.use(authenticationMiddleware())
```

Comment:
@jonhindle this here breaks if i remove the invoking parens. When I refactored the the `authenticationMiddleware` I removed the wrapping function, but it will not work like this. Any ideas?

Reply:
authenticationMiddleware returns a function that when called returns middleware.
```
function authenticationMiddleware() {
  // eslint-disable-next-line
  return (req, res, next) => {
```
I can see no reason why the middleware requires wrapping though. We're not passing any vars into it. Just return a middleware function. No need to create it every time?
```
function authenticationMiddleware(req, res, next) => {
```

Reply:
Sorry I misunderstood the query. Test related. All sorted.